src/stl/chain.rs

```diff
@@ -40,4 +43,18 @@ impl ProofOfReserves {
     pub fn new(utxo: Outpoint, proof: SmallBlob) -> ProofOfReserves {
         ProofOfReserves { utxo, proof }
     }
+
+    pub fn from_strict_val_unchecked(value: &StrictVal) -> Self {
+        let utxo = Outpoint::from_str(&value.unwrap_struct("utxo").unwrap_string())
+            .expect("invalid outpoint");
+        let proof = SmallBlob::from_collection_unsafe(
+            value
+                .unwrap_struct("proof")
+                .unwrap_bytes()
+                .try_into()
+                .expect("invalid data"),
+        );
```

Comment:
Fixes linter warning:

```
use of a fallible conversion when an infallible one could be used
for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#unnecessary_fallible_conversions
`#[warn(clippy::unnecessary_fallible_conversions)]` on by default
```

```suggestion
        let proof =
            SmallBlob::from_collection_unsafe(value.unwrap_struct("proof").unwrap_bytes().into());
```